static/js/app.js

```diff
@@ -37,139 +36,31 @@ async function loadAboutMePage() {
   document.getElementById('app').insertAdjacentHTML('beforeend', aboutMeHtml);
 };
 
-//********** load experience page **********\\     
-async function loadExperiencePage() {
+//********** load event html**********\\     
+async function loadEventHtml(eventType) {
   //gets experience html view
-  const response = await fetch('./static/html/Experience.html');
+  const response = await fetch(`./static/html/${eventType}.html`);
   if (!response.ok) {
-      throw new Error(`Failed to fetch Experience.html: ${response.status}`);
+    throw new Error(`Failed to fetch ${eventType}.html: ${response.status}`);
   }
   //awaits the fectch response if it is successfull
-  const experienceHtml = await response.text();
+  const eventHtml = await response.text();
   //Append the fectched html content to 'app' div
-  document.getElementById('app').insertAdjacentHTML('beforeend', experienceHtml);
+  document.getElementById('app').insertAdjacentHTML('beforeend', eventHtml);
 }
 
-async function addExperienceEvents() {
+//********** add events to html**********\\     
```

Comment:
I would add a space before the second sequence of `*`, just for symmetry ;)

Reply:
check other occurrences

Reply:
Fixed.